code/index.html

```diff
@@ -1,30 +1,324 @@
+        </p>
       </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <button class="subtitle faq__question" id="question5">
+        Help! I'm a victim...
+      </button>
+      <div class="faq__answer">
+        <p class="text faq__text">
+          Our deepest apologies for your misfortune. 
+          <br>Regrettably, this offensive incursion is both anonymous and irreversible.
+          <br><br>We do have some advice that may be useful:
+          <br>• Manually unsubscribe from the newsletters.
+          <br>• Create a new email.
+          <br>• Change your phone number.
+          <br>• Be nicer to people.
+          <br>• Find some new "friends".
+          <br>• Take some time off to reflect on why this might have happened to you.
+          <br><br><em>"Before I was afflicted I went astray: but now have I kept my word."</em> <strong>Psalm 119:67</strong>
       </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
+
+    </div>
+
+    <!-- Signup Form Container -->
+    <div class="block-container signup" id="signup">
+      <!-- Form Header -->
+      <img class="signup__image" src="images/death-5.png" alt="Death Note" />
+      <h1 class="title signup__title">
+        R.I.P. INBOX
+      </h1>
+      <!-- Form -->
+      <form id="form" class="signup__form" action="http://httpbin.org/anything" method="post">
+
+        <!-- Name, Email, Phone -->
+        <label for="name">
+          <input name="name" id="name" type="text" placeholder="Their name" required />
+        </label>
+        <label for="email">
+          <input name="email" id="email" type="email" placeholder="Their email" required />
+        </label>
+        <label for="phone">
+          <input name="phone" id="phone" type="tel" placeholder="Their phone number (Optional)" />
+        </label>
+
+        <!-- Radio Buttons -->
+        <!-- Email Only Button -->
+        <div class="select-container">
+          <div class="select-container__object">
+            <label for="emailOnly" class="radio--remove">
+              <input type="radio" id="emailOnly" name="mailOrTele" value="Email only" required />
+              <span class="radio--custom"></span>
+          </div>
+          <div class="select-container__text">
+            <span class="signup__options">
+              Email only
+            </span>
+            </label>
+          </div>
+        </div>
+        <!-- Email and Telemarketing Button -->
+        <div class="select-container">
+          <div class="select-container__object">
+            <label for="emailTelemarketing" class="radio--remove">
+              <input type="radio" id="emailTelemarketing" name="mailOrTele" value="Email and Telemarketing" />
+              <span class="radio--custom"></span>
```

Comment:
This part of remove and custom the radio buttons seems a bit complicated.
Im impressed you even started that "project".

Reply:
It was honestly horrifying! A real struggle 😨